tests/tests/swfs/avm2/id3_info/test.as

```diff
@@ -0,0 +1,24 @@
+package
+{
+	import flash.display.Sprite;
+	import flash.events.Event;
+	import flash.media.Sound;
+	import flash.net.URLRequest;
+	
+	public class Main extends Sprite
+	{
+		
+		public function Main()
+		{
+			var sound:Sound = new Sound();
+			sound.addEventListener(Event.ID3, function(event:Event):void{
+				trace("id3event id3:" + sound.id3);
+				var properties:Array = ["album", "artist", "comment", "genre", "songName", "track", "year"];
+				for (var i:String in properties) trace(properties[i]+":"+sound.id3[properties[i]]);
+			});
+			sound.load(new URLRequest("file:///test_audio.mp3"));
```

Comment:
I think this can just be `new URLRequest("test_audio.mp3")`, no?

Reply:
I just follow this: https://github.com/ruffle-rs/ruffle/blob/master/tests/framework/src/backends/navigator.rs#L243

Reply:
>`Url::parse("file:///")?;`

This means that there is an assumed base URL of `file:///`, not that it needs to start with `file:///`.

Reply:
Does `new URLRequest("test_audio.mp3")` not work?

Reply:
It works, there was a misoperation that makes them unchanged.